aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
@Rick-Anderson @blowdart I've got the changes committed to [my fork here](https://github.com/joeaudette/Docs/tree/ra/secure-user-data2/aspnetcore/security/authorization/secure-data/samples/final15) ready to make a new pull request I think, if I have understood the feedback correctly